Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "system76-firmware"
-version = "1.0.44"
+version = "1.0.45"
```

Comment:
`Cargo.lock` needs to be updated as well.

Reply:
Thanks, fixed